tests/create_LWE.py

```diff
@@ -0,0 +1,231 @@
+
+    unit_operation.discretization.ncol = n_col
+    unit_operation.discretization.npar = n_par
+    unit_operation.discretization.par_disc_type = 'EQUIDISTANT_PAR'
+    unit_operation.discretization.use_analytic_jacobian = not ad_jacobian
+    unit_operation.discretization.max_krylov = 0
+    unit_operation.discretization.gs_type = True
+    unit_operation.discretization.max_restarts = 10
+    unit_operation.discretization.schur_safety = 1e-8
+    unit_operation.discretization.weno = {'boundary_model': 0, 'weno_eps': 1e-10, 'weno_order': 3}
+
+def configure_particles(unit_operation):
+    par_radius = 4.5e-5
+    par_porosity = 0.75
+
+    unit_operation.particle_radius = par_radius
+    unit_operation.particle_porosity = par_porosity
+
+    unit_operation.discretization.par_geom = 'SPHERE'
+ 
+
+def get_binding_model_class(binding_type, component_system):
+
+    binding_classes = {
+    'linear': Linear,
+    'langmuir': Langmuir,
+    'langmuir_ldf': LangmuirLDF,
+    'bilangmuir': BiLangmuir,
+    'bilangmuir_ldf': BiLangmuirLDF,
+    'freundlich_ldf': FreundlichLDF,
+    'steric_mass_action': StericMassAction,
+    'anti_langmuir': AntiLangmuir,
+    'spreading': Spreading,
+    'self_association': SelfAssociation,
+    'bi_steric_mass_action': BiStericMassAction,
+    'multistate_steric_mass_action': MultistateStericMassAction,
+    'simplified_multistate_steric_mass_action': SimplifiedMultistateStericMassAction,
+    'saska': Saska,
+    'generalized_ion_exchange': GeneralizedIonExchange
+    }
+
+    try:
+        binding_class = binding_classes[binding_type]
+        return binding_class(component_system)
+    except KeyError:
+        raise ValueError(f'Unknown binding type {binding_type}')
+
+
+def configure_adsorption(unit_operation, component_system, binding_type, is_kinetic):
+    
+    kA = 35.5
+    kD = 1000.0
+    nu = 4.7
+    sigma = 11.83
+    sma_lambda = 1.2e3
+
+    binding_type = 'steric_mass_action'
+    binding_model = get_binding_model_class(binding_type, component_system)
+
+    binding_model.is_kinetic = is_kinetic
+
+    unit_operation.binding_model = binding_model
+    unit_operation.n_bound_states = [1] * 4 #n_comp
+    unit_operation.adsorption_rate = kA
+    unit_operation.desorption_rate = kD
+    unit_operation.characteristic_charge = nu
+    unit_operation.steric_factor = sigma
+    unit_operation.capacity = sma_lambda
+
+def configure_film_diffusion(unit_operation, n_comp, n_partype):
+    film_diff = [6.9e-6] * n_comp
+    ("FILM_DIFFUSION_MULTIPLEX", 0)
+    writer.vector("FILM_DIFFUSION", n_comp, film_diff)
+
+def configure_pore_diffusion(unit_operation, n_comp, n_partype):
+    par_diff = [7e-10, 6.07e-11, 6.07e-11, 6.07e-11]
+    writer.vector("PAR_DIFFUSION", n_comp, par_diff)
+
+def configure_flow_direction(unit_operation, reverse_flow):
+    unit_operation.flow_direction = -1 if reverse_flow else 1
+
+def configure_unit_solver(writer):
+
+    ("MAX_KRYLOV", 0)
```

Comment:
Note that these parameters already have good default values in the `Cadet` class. We still might want to overwrite certain options but for an MVP, I think it's fine to just use them "as is".